### PR TITLE
Added ARM template for Service Bus namespace queue, topic, subscripti…

### PR DESCRIPTION
**Pull request by shriazure**
…ons, filters and SAS keys creation

### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by shriazure** [comment]
ARM template to create Service Bus Namespace, Queue, Topic, Subscriptions, Filters and SAS Keys for Send and Listen operations.

File: .gitignore [hunk 1]
```diff
@@ -30,4 +30,5 @@ node_modules
 Nese_OMS_NEstedforOMSWS.json
```
Comment (bmoore-msft) [inline .gitignore]: remove this file from the pr

File: 101-servicebus-queue-topic-subscription-filter-saskeys/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+    "sendOnlySASKeyName": "[parameters('sasPolicyNameToSend')]",
+    "authRuleResourceIdForQueue-Send": "[resourceId('Microsoft.ServiceBus/namespaces/queues/authorizationRules', parameters('serviceBusNamespaceName'), parameters('serviceBusQueueName'), variables('sendOnlySASKeyName'))]",
+    "authRuleResourceIdForTopic-Send": "[resourceId('Microsoft.ServiceBus/namespaces/topics/authorizationRules', parameters('serviceBusNamespaceName'), parameters('serviceBusTopicName'), variables('sendOnlySASKeyName'))]",
+    "listenOnlySASKeyName": "[parameters('sasPolicyNameToListen')]",
+    "authRuleResourceIdForQueue-Listen": "[resourceId('Microsoft.ServiceBus/namespaces/queues/authorizationRules', parameters('serviceBusNamespaceName'), parameters('serviceBusQueueName'), variables('listenOnlySASKeyName'))]",
+    "authRuleResourceIdForTopic-Listen": "[resourceId('Microsoft.ServiceBus/namespaces/topics/authorizationRules', parameters('serviceBusNamespaceName'), parameters('serviceBusTopicName'), variables('listenOnlySASKeyName'))]",
+    "sbVersion": "2017-04-01"
```
Comment (bmoore-msft) [inline 101-servicebus-queue-topic-subscription-filter-saskeys/azuredeploy.json]: remove this and use a literal apiVersion

File: 101-servicebus-queue-topic-subscription-filter-saskeys/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      "value": "GEN-UNIQUE"
+    },
+    "serviceBusTopicName": {
+      "value": "GEN-UNIQUE"
+    },
+    "serviceBusTopicSubscriptionName": {
+      "value": "GEN-UNIQUE"
+    },
+    "serviceBusTopicSubscriptionSqlFilter": {
+      "value": "MessageType = 'type01'"
```
Comment (bmoore-msft) [inline 101-servicebus-queue-topic-subscription-filter-saskeys/azuredeploy.parameters.json]: use defaultValues for any non-GEN values

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can address the feedback.

**Pull request closed without merging**